frontend/app/quest/Quest_Activity.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+    <!-- to do:  update central stylesheet to this style sheet code.-->
+    
+<html ng-app="quest_builder">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Hello">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="shortcut icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <title>PlanetLAB> Quest Builder</title>
+        <link rel="stylesheet" href="css/style.css" type="type/css" />
+        </head>
+
+    <body>
+    <div ng-view></div>
+        <img src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games">
+        <ol class="controlNav" >
```

Comment:
These should all be ui-srefs, and each of the following views should be set as states in the routing, as we went over a while ago. We should never be linking to raw html files